app/reducers/__tests__/events.spec.ts

```diff
@@ -1,241 +1,44 @@
-// Hack because we have circular dependencies
-// (companies -> events -> index -> frontpage -> events)
-// This import resolves dependencies properly..
 import 'app/reducers';
```

Comment:
Is the import still needed though?

Reply:
Nope, I think I must have inteded to remove it along with the comment👍